$MirOS: ports/devel/automake/1.9/patches/patch-lib_am_lisp_am,v 1.1.7.1 2005/03/18 15:44:28 tg Exp $
--- lib/am/lisp.am.orig	Sat May 14 20:20:44 2005
+++ lib/am/lisp.am	Sun Oct  2 11:37:00 2005
@@ -86,7 +86,7 @@ am__installdirs += "$(DESTDIR)$(%NDIR%di
 ?!BASE?%DIR%LISP_INSTALL = $(install_sh_DATA)
 ?EXEC?.PHONY install-exec-am: install-%DIR%LISP
 ?!EXEC?.PHONY install-data-am: install-%DIR%LISP
-install-%DIR%LISP: $(%DIR%_LISP) $(ELCFILES)
+install-%DIR%LISP:
 	@$(NORMAL_INSTALL)
 ## Do not install anything if EMACS was not found.
 	@if test "$(EMACS)" != no; then \
